meta-balena-common/recipes-core/os-config/os-config_1.2.9.bb

```diff
@@ -251,9 +262,12 @@ SRC_URI += " \
     crate://crates.io/winreg/0.6.2 \
     crate://crates.io/winutil/0.1.1 \
     crate://crates.io/ws2_32-sys/0.2.1 \
+    git://github.com/balena-os/fatrw;protocol=https;nobranch=1;name=fatrw;destsuffix=fatrw \
 "
 
-
+SRCREV_FORMAT .= "_fatrw"
+SRCREV_fatrw = "v0.1.10"
```

Comment:
@majorz This needs to be a sha1

Reply:
@alexgg done and awaiting on the CI checks to finish